system-x/services/ssh/src/main/java/software/tnb/ssh/resource/local/LocalSshServer.java

```diff
@@ -0,0 +1,63 @@
+package software.tnb.ssh.resource.local;
+
+import software.tnb.common.deployment.Deployable;
+import software.tnb.ssh.service.SshServer;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.auto.service.AutoService;
+
+@AutoService(SshServer.class)
+public class LocalSshServer extends SshServer implements Deployable {
+
+    private static final Logger LOG = LoggerFactory.getLogger(LocalSshServer.class);
+
+    private SshServerContainer container;
+
+    @Override
+    public void deploy() {
+        LOG.info("Starting SSH Server container");
+        container = new SshServerContainer(image(), containerEnvironment());
+        container.start();
+        LOG.info("SSH Server container started");
+    }
+
+    @Override
+    public void undeploy() {
+        if (container != null) {
+            LOG.info("Stopping SSH Server container");
+            container.stop();
+        }
+    }
+
+    @Override
+    public void openResources() {
+
+    }
+
+    @Override
+    public void closeResources() {
+
+    }
+
+    @Override
+    public String username() {
+        return "redhat";
+    }
+
+    @Override
+    public String host() {
+        return "0.0.0.0";
```

Comment:
why not container.getHost() ? instead of 0.0.0.0, in case of sidecar/remote docker host there may be issues

Reply:
After some tests container.getHost() seems working fine.